plugins/language_rust.lua

```diff
@@ -15,6 +15,7 @@ syntax.add {
     { pattern = "-?%d+[%d%.eE]*f?",     type = "number"   },
     { pattern = "-?%.?%d+f?",           type = "number"   },
     { pattern = "[%+%-=/%*%^%%<>!~|&]", type = "operator" },
+    { pattern = "[%a_][%w_]*!%f[%[(]",  type = "function" },
```

Comment:
Is this change to the Rust plugin intended?

Reply:
Yes. It is for highlighting macros. https://github.com/rxi/lite-plugins/pull/128

Reply:
As this was about the Julia plugin it seemed a little strange, but that's ok.